lib/metric_fu/metrics/reek/generator.rb

```diff
@@ -1,6 +1,6 @@
 module MetricFu
   class ReekGenerator < Generator
-    REEK_REGEX = /^(\S+) (.*) \((.*)\)$/
+    REEK_REGEX = /^\s*\[([^:]+)\]:(\S+)\s+(.*)\s+\((.*)\)$/
```

Comment:
I would say ```/^\s*\[(\d+), \d+\]:(\S+)\s+(.*)\s+\((.*)\)$/``` is a better matcher, that way we can filter out the line number directly later on.